demisto_sdk/commands/format/update_incidentfields.py

```diff
@@ -17,8 +17,8 @@ class IncidentFieldJSONFormat(BaseUpdateJSON):
             output (str): the desired file name to save the updated version of the JSON to.
     """
 
-    def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '', no_validate: bool = False):
-        super().__init__(input, output, path, from_version, no_validate)
+    def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '', no_validate: bool = False, verbose: bool = False):
+        super().__init__(input, output, path, from_version, no_validate, verbose)
```

Comment:
keyword argument shouldn't be passed as positional argument (relevant for all appearances)